src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+  };
+
+  _PVector.random2D = function(target, parent) {
+    //const rnd = parent? parent : target instanceof Processing? target : Math;
+    const rnd = parent? parent : target && target.random? target : Math;
+    return _PVector.fromAngle(TAU * rnd.random(), target);
+  };
+
+  _PVector.random3D = function(target, parent) {
+    //const rnd = parent? parent : target instanceof Processing? target : Math,
+    const rnd = parent? parent : target && target.random? target : Math,
+          ang = TAU * rnd.random(),
+          vz  = 2 * rnd.random() - 1,
+          vzr = Math.sqrt(1 - vz*vz),
+          vx  = vzr * Math.cos(ang),
+          vy  = vzr * Math.sin(ang);
+    return target? target.set(vx, vy, vz) : new _PVector(vx, vy, vz);
+  };
+
+  _PVector.dist = function(v1, v2) {
+    return Math.sqrt(_PVector.distSq(v1, v2));
+  };
+
+  _PVector.distSq = function(v1, v2) {
+    return sq(v1.x - v2.x) + sq(v1.y - v2.y) + sq(v1.z - v2.z);
+  };
+
+  _PVector.dot = function(v1, v2) {
+    return v1.x*v2.x + v1.y*v2.y + v1.z*v2.z;
+  };
+
+  _PVector.cross = function(v1, v2, target) {
+    const cx = v1.y*v2.z - v2.y*v1.z,
+          cy = v1.z*v2.x - v2.z*v1.x,
+          cz = v1.x*v2.y - v2.x*v1.y;
+    return target? target.set(cx, cy, cz) : new _PVector(cx, cy, cz);
+  };
+
+  _PVector.angleBetween = function(v1, v2) {
+    if (!v1.x && !v1.y && !v1.z || !v2.x && !v2.y && !v2.z) { return 0; }
+    const amt = _PVector.dot(v1, v2) / Math.sqrt(v1.magSq() * v2.magSq());
+    return amt <= -1? Math.PI : amt >= 1? 0 : Math.acos(amt);
+  };
+
+  _PVector.lerp = function(v1, v2, amt) {
+    return v1.copy().lerp(v2, amt);
+  };
+
+  _PVector.add = function(v1, v2, target) {
+    return target? target.set(v1.x + v2.x, v1.y + v2.y, v1.z + v2.z)
+                 : new _PVector(v1.x + v2.x, v1.y + v2.y, v1.z + v2.z);
+  };
+
+  _PVector.sub = function(v1, v2, target) {
+    return target? target.set(v1.x - v2.x, v1.y - v2.y, v1.z - v2.z)
+                 : new _PVector(v1.x - v2.x, v1.y - v2.y, v1.z - v2.z);
+  };
+
+  _PVector.mult = function(v, n, target) {
+    if (typeof n === 'number') {
+      return target? target.set(v.x*n, v.y*n, v.z*n)
+                   : new _PVector(v.x*n, v.y*n, v.z*n);
+    }
+    return target? target.set(v.x*n.x, v.y*n.y, v.z*n.z)
+                 : new _PVector(v.x*n.x, v.y*n.y, v.z*n.z);
+  };
+
+  _PVector.div = function(v, n, target) {
+    if (typeof n === 'number') {
+      return target? target.set(v.x/n, v.y/n, v.z/n)
+                   : new _PVector(v.x/n, v.y/n, v.z/n);
+    }
+    return target? target.set(v.x/n.x, v.y/n.y, v.z/n.z)
+                 : new _PVector(v.x/n.x, v.y/n.y, v.z/n.z);
+  };
+
+  _PVector.prototype = {
+    copy: function() {
+      return new _PVector(this.x, this.y, this.z);
+    },
+    get: function(target) {
+      if (!arguments.length) { return this.copy(); } // @Deprecated
+      if (typeof target !== 'object') { return this.array(); }
```

Comment:
> What he and his team do in p5.js has no true relevance for Processing.js, since the projects are so different...

All of those mentions above are proofs I've collected to solve your disbelief about **get()** being  deprecated for Java Mode!
It got nothing to do w/ differences between pjs & p5.js. Dunno why you brought that issue too!

Please pay attention that @Shiffman mentioned Processing, not p5.js here:

> Also, we changed **get()** to **copy()** in Processing so I'm doing that here as well.